docs/docusaurus/docs/cloud/try_gx_cloud.md

```diff
@@ -21,6 +21,12 @@ If you've tested GX Cloud features and functionality and discovered it's a great
 
 - You have a [Docker instance](https://docs.docker.com/get-docker/).
 
+## Exploratory deployment pattern
```

Comment:
```suggestion
## Test deployment pattern
```
I'm trying to find a simpler word than _exploratory_.  I am open to suggestions!